source/fundamentals/logging.txt

```diff
@@ -362,7 +342,7 @@ a third-party logging package, into your application.
 
    .. step:: Set the logger as the ``Sink`` for your ``Client``.
 
-      In this example, the logger logs against commands at the ``DEBUG`` level.
+      In this example, the logger logs against commands at the ``LevelDebug`` level.
```

Comment:
Suggestions:

- I think "this example" is ambiguous because it is used to refer to the entirety of the section at the start. Instead, refer to the snippet.

- I think the explanation could be more accurate since it doesn't actually log anything in the code snippet, only sets it up.

- I think "against" should be omitted since "logs"sufficiently describes what the logger does to commands. Applies to all instances.

"In the following code example, the logger is configured to log commands at the LevelDebug level."